src/Discovery/Discovery.css

```diff
@@ -49,21 +50,34 @@
   align-items: center;
 }
 
+.discovery-header__stats-wrapper {
+  display: inline-block;
+}
+
+.discovery-header__stats-wrapper#Studies {
+  width: 65px;
+}
+
+.discovery-header__stats-wrapper#TotalSubjects {
```

Comment:
The class before the id selector gives us some context, but it's moot due to the id. Do we need to bother with the BEM style selector? Really I'd rather not see us use ids, but either way seems redundant.